server/src/test/java/org/opensearch/index/shard/RemoteStoreRefreshListenerTests.java

```diff
@@ -282,7 +282,6 @@ public void testRefreshSuccessOnSecondAttempt() throws Exception {
     /**
      * Tests retry flow after snapshot and metadata files have been uploaded to remote store in the failed attempt.
      * Snapshot and metadata files created in failed attempt should not break retry.
-     * @throws Exception
```

Comment:
looks not intentional?

Reply:
Intentional. Build is throwing warnings due to this so removed.